pytest_catchlog.py

```diff
@@ -139,10 +180,13 @@ def emit(self, record):
 class CatchLogFuncArg(object):
```

Comment:
I decided to pick `LogCaptureFixture` since the word _catch_ sounds a bit weird for me.  :airplane: :arrow_left: :arrow_right: